backend/src/main/java/net/gunn/elimination/model/Kill.java

```diff
@@ -0,0 +1,49 @@
+package net.gunn.elimination.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+
+@Entity
+public class Kill implements Serializable {
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@JsonIgnore
+	private String id;
+
+	@JsonProperty
+	private String eliminator;
```

Comment:
this should be a EliminationUser with a one-to-many mapping

Reply:
but i thought you said to store them as strings :anguish:

Reply:
yeah, but not for the DB model objects

Reply:
we should be passing around strings betwene clients, but the DB model objects should have all of the join relationships encoded in them